Makefile

```diff
@@ -1,29 +1,25 @@
-GOPACKAGES := $(shell go list github.com/senseyeio/mbgo/...)
+GOPACKAGES := $(shell go list ./...)
 
```

Comment:
In addition to my comment above about removing all these checks from here and making it run in a docker, I think this makefile needs some changes:
- there is no default target, so when you run `make` it runs `make errcheck` which is probably not what people wants
- the integration tests should probably run whatever is required for it to work. If I run it manually without a mountebank instance running, the tests still pass, which is weird as the README explains they rely on some mountebank running.

Reply:
- I've added a `default` target to the Makefile which runs the `fmt`, `unit` and `lint` targets.

Reply:
I'm sorry, I don't know what happened there, I cannot reproduce it either, maybe I didn't notice I had a mountebank instance running.